ZeldaFullEditor/GFX.cs

```diff
@@ -1,15 +1,8 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
-using System.Diagnostics;
 using System.Drawing;
 using System.Drawing.Imaging;
-using System.Linq;
 using System.Runtime.InteropServices;
 using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Forms;
 using ZeldaFullEditor.Properties;
 
 // TODO clean up all the magic numbers
```

Comment:
nit: vague todo. are we removing them? replacing with local constants? moving to constants.cs?

Reply:
moving to the constants.cs